test/system/505-networking-pasta.bats

```diff
@@ -789,3 +775,19 @@ EOF
     CONTAINERS_CONF_OVERRIDE=$containersconf run_podman run --net=pasta:--ns-mac-addr,"$mac2" $IMAGE ip link show myname
     assert "$output" =~ "$mac2" "mac address from cli is set on custom interface"
 }
+
+### Rootless unshare testins
+
+@test "Podman unshare --rootless-netns with Pasta" {
+    skip_if_remote "unshare is local-only"
+
+    pasta_iface=$(default_ifname)
+
+    run_podman unshare --rootless-netns ip addr
+    is "$output" ".*${pasta_iface}.*"
+
+    # To prevent a leak of a Pasta process, start a container to cause it to be
+    # cleaned up.
+
+    run_podman run --network bridge $IMAGE true
```

Comment:
This is not needed anymore, the logic for this was a old relict from before the c/common rework and should have been removed by me before that.

Or better just remove the test completely it doesn't add anything compared to the `podman rootless-netns pasta processes should be in different cgroup` test